web/app/themes/radcliffe/report.php

```diff
@@ -66,4 +63,4 @@
 
 </div>
 
-<?php get_footer(); ?>
+<?php get_footer(); ?>
```

Comment:
Every PHP file should have 1 empty line at the end of the file. Can this please be added? Thank you :)

Reply:
Ok.